apps/journeys-admin/src/components/ReportButtons/ReportButtons.spec.tsx

```diff
@@ -0,0 +1,55 @@
+import { render, fireEvent,waitFor } from '@testing-library/react'
+import { MockedProvider } from '@apollo/client/testing'
+import { SnackbarProvider } from 'notistack'
+import { FlagsProvider } from '@core/shared/ui/FlagsProvider'
+import { JourneyProvider } from '@core/journeys/ui/JourneyProvider'
+import { ReportButtons } from './ReportButtons'
```

Comment:
```suggestion
import { ReportButtons } from './ReportButtons'
import { NextRouter, useRouter } from 'next/router'
```

import router and type